pub_sub/python/sdk/README.md

```diff
@@ -42,7 +42,38 @@ sleep: 10
 -->
 
 ```bash
-dapr run --app-id order-processor-sdk --resources-path ../../../components/ --app-port 6001 -- uvicorn app:app --port 6002
+dapr run --app-id order-processor-sdk --resources-path ../../../components/ --app-port 6002 -- uvicorn app:app --port 6002
+```
+
+<!-- END_STEP -->
+
+2. Run the Python subscriber app (fastapi version) with Dapr: 
+
+<!-- STEP
+name: run
+-->
+
+```bash
+cd ./order-processor-fastapi
+pip3 install -r requirements.txt
+```
+
+<!-- END_STEP -->
+
+<!-- STEP
+name: Run python subscriber
+expected_stdout_lines:
+  - '== APP == Subscriber received : 4'
+  - "Exited App successfully"
+expected_stderr_lines:
+output_match_mode: substring
+working_dir: ./order-processor-fastapi
+background: true
+sleep: 10
+-->
+
+```bash
+dapr run --app-id order-processor-sdk-fastapi --resources-path ../../../components/ --app-port 6003 -- uvicorn app:app --port 6003
```

Comment:
I don't see any app defined to run on port 6003;   app.py in order-processor-fastapi should have app defined on port 6003.

Reply:
> I don't see any app defined to run on port 6003; app.py in order-processor-fastapi should have app defined on port 6003.

The port is defined by `uvicorn app:app --port 6003` part.